dex/msgjson/types.go

```diff
@@ -845,9 +841,18 @@ func (c *Connect) Serialize() []byte {
 	return append(s, uint64Bytes(c.Time)...)
 }
 
+// Order describes an order, its status and fill, and is returned as part of a
+// ConnectResult or as part of a OrderStatusResult.
+type Order struct {
```

Comment:
I wanted to avoid `OrderStatusResult` because the `connect` response returns this type as well. Now that I read your comment, I think `OrderStatus` works fine. The `OrderStatusResult` is def. not necessary. Added that just to read better.